Test failure: nodeos_read_terminate_at_block_if_lr_test

https://github.com/AntelopeIO/spring/actions/runs/16200499530/job/45738833997

```
Traceback (most recent call last):
  File "/__w/spring/spring/build/tests/nodeos_read_terminate_at_block_test.py", line 363, in <module>
    assert cluster.getNode(nodeId).waitForLibToAdvance(), f"LIB did not advance for {nodeId}"
AssertionError: LIB did not advance for 8
```

--- a/plugins/net_plugin/net_plugin.cpp
+++ b/plugins/net_plugin/net_plugin.cpp
@@ -2400,7 +2400,7 @@ namespace eosio {
       {
          // reset sync on rejected block
          fc::lock_guard g( sync_mtx );
-         if (blk_num <= sync_next_expected_num-1) { // no need to reset if we already reset and are syncing again
+         if (sync_last_requested_num != 0 && blk_num <= sync_next_expected_num-1) { // no need to reset if we already reset and are syncing again
             sync_last_requested_num = 0;
             sync_next_expected_num = my_impl->get_fork_db_root_num() + 1;
          }

